appveyor.yml

```diff
@@ -46,11 +46,9 @@ install:
 build: false
 
 test_script:
-  # Change to a non-source folder to make sure we run the tests on the
-  # installed library.
-  # TODO: Once the tests will be moved we have to revert this part
   # - "cd C:\\"
```

Comment:
be sure to get outside of the folder for testing the install package and not the package itself